application/cms/page_service.py

```diff
@@ -101,6 +111,11 @@ def update_page(self, page, data, last_updated_by, data_source_forms=None):
                     page.parent = new_subtopic
                     page.position = len(new_subtopic.children)
 
+                    measure.subtopics = [
+                        Subtopic.query.filter_by(uri=new_subtopic.uri, position=new_subtopic.position).one()
```

Comment:
Seems weird to include the position in the query?  I'd have expected a combination of uri + topic_id?

Reply:
That makes more sense. 👍

Reply:
Done!